doc/api/deprecations.md

```diff
@@ -1006,11 +1006,11 @@ URL parser that requires strictly valid URLs. Passing an invalid URL is
 deprecated and support will be removed in the future.
 
 <a id="DEP0110"></a>
-### DEP0110: _getActiveHandles and _getActiveRequests`
+### DEP0110: _getActiveHandles and _getActiveRequests
 
 Type: Runtime
 
-As `getActiveHandles()` and `getActiveRequests()` are getting officially
+As `process.getActiveHandles()` and `process.getActiveRequests()` are getting officially
```

Comment:
Nit: please wrap long lines at 80 characters to prevent doc linting issues.

Reply:
Fixed !